wp-admin/js/widgets/media-image-widget.js

```diff
@@ -74,125 +74,6 @@
 		},
 
 		/**
-		 * Open the media select frame to chose an item.
-		 *
-		 * @param {jQuery.Event} event - Event.
-		 * @returns {void}
-		 */
-		selectMedia: function selectMedia( event ) {
-			var control = this, selection, mediaFrame, CustomizedDisplaySettingsLibrary, customizedDisplaySettings;
-
-			event.preventDefault();
```

Comment:
We'll continue to need this so clicking the media library link in the error message doesn't actually take you to the media library